### ISSUE TITLE
Word 2016 crashes with NVDA

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#6567]
Since last month or so, it happens quite often that Word 2016 crashes on multiple PC's with Windows 10 X64 when I open a document. It doesn't seem to happen in the case of a blank document.
I've tried to generate a crash dump using focus.appModule.dumpOnCrash(), but strange enough, no dump is generated even though Word crashes.

I've done tests with JAWS 17 as well, but am unable to make it crash

[Here is a log](https://github.com/nvaccess/nvda/files/594190/wordcrash.txt)

**Comment by leonardder** [comment]
It seems that I forgot one thing, repairing my office. This seems to fix my particular issue.
